samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/GeometrySelection.razor

```diff
@@ -0,0 +1,80 @@
+﻿@page "/geometry-selection"
+@using dymaptic.GeoBlazor.Core.Model.EventArgs
+
+<h3>GraphicSelection</h3>
+<MapView Longitude="_longitude"
+         Latitude="_latitude"
+         Style="height: 600px; width: 100%;">
+    <Map ArcGISDefaultBasemap="arcgis-topographic">
+        <GraphicsLayer>
+            @foreach (var feature in Features)
+            {
+                <Graphic Attributes="feature.Attributes">
+                    <Point OnGeometrySelected="GeometrySelected" Longitude="feature?.Point?[0]" Latitude="feature?.Point?[1]" />
+                    <SimpleMarkerSymbol Color="@(new MapColor(81, 46, 132))">
+                        <Outline Color="@(new MapColor(255, 255, 255))" Width="1" />
+                    </SimpleMarkerSymbol>
+                </Graphic>
+            }
+        </GraphicsLayer>
+    </Map>
+</MapView>
+
+@code {
+
+    double _latitude = 34.027;
+    double _longitude = -118.805;
+
+    List<MyObject> Features { get; set; } = GetFeatures();
+
+    Task GeometrySelected(GeometrySelectedEventArgs<Geometry> e)
+    {
+        return Task.CompletedTask;
```

Comment:
What is the ultimate use case for the samples page? Will we display something that says "Geometry x was clicked!"?